src/main/menu.ts

```diff
@@ -26,7 +26,7 @@ export function showAbout(browserWindow: BrowserWindow) {
     `Copyright 2020 Mirantis, Inc.`,
   ]
   dialog.showMessageBoxSync(browserWindow, {
-    title: `${isWindows ? " ".repeat(2) : ""}${appName}`,
+    title: `${" ".repeat(isWindows as any * 2)}${appName}`,
```

Comment:
This is totally not obvious: `isWindows as any * 2`, please revert.

Reply:
oops, I should try to have better commit control. I was seeing if this would work is JS since it is a very python-esk type of operation.